lib/Page.js

```diff
@@ -62,10 +63,43 @@ function calculateNewBaseUrl(filePath, root, lookUp) {
   return calculate(filePath, []);
 }
 
+function insertFlexDiv(pageData) {
+  const $ = cheerio.load(pageData);
+  const footers = $('footer');
+
+  footers.each((index, footer) => {
```

Comment:
If we are only using one footer in the end, seems a bit redundant to add flexDiv to all footers. :/